sentry/templates/configmap-sentry.yaml

```diff
@@ -352,113 +356,47 @@ data:
     ############
     # Features #
     ############
-
-
-    SENTRY_FEATURES = {
-      "auth:register": {{ .Values.auth.register | ternary "True" "False" }}
-    }
+    
```

Comment:
primary change. updating certain values rather than overwriting the array

[reference](https://github.com/getsentry/self-hosted/blob/master/sentry/sentry.conf.example.py#L253)